Gormsson/Source/GattCharacteristic/TemperatureMeasurement.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  TemperatureMeasurement.swift
```

Comment:
The file name should be: HealthThermometer.swift
In futur it will also handle Temperature Type, Intermediate Temperature and Measurement Interval.